investd/common.py

```diff
@@ -0,0 +1,38 @@
+from datetime import datetime
+from enum import Enum
+
+from pydantic.dataclasses import dataclass
+
+
+class Currency(Enum):
```

Comment:
After moving those, shouldn't you drop the model.py file?

Reply:
That's right. I forgot about it. I just removed it now